test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -215,7 +215,7 @@ BOOST_AUTO_TEST_CASE(dev_desc_after_nl)
 	char const* natspec = "{"
 	"\"methods\":{"
 	"    \"mul(uint256,uint256)\":{ \n"
-	"        \"details\": \" Multiplies a number by 7 and adds second parameter\",\n"
+	"        \"details\": \"Multiplies a number by 7 and adds second parameter\",\n"
```

Comment:
This is not part of the result of the detailed description of `mul`. It's simply the json representation separating one dictionary key from the next.